platform/podjs/README.md

```diff
@@ -29,12 +29,102 @@ soon.
 
 Simply run `build.js` in the repository root.
 
-## Usage
+## How to setup a feature to build with podjs
 
-You will need to use this module for creating a polyPod-compatible
-feature. Follow these instructions:
+To create a polyPod compatible feature, you will need to use this module.
+Follow these instructions to do so:
 
-1. Add _podjs_ as a development dependency to your feature.
-2. Copy or link `node_modules/@polypoly-eu/podjs/dist/pod.js`.
-3. Include `pod.js` from your feature's main document - before the
-   feature's own code.
+1.  In order to point to `podjs` from the polypoly-eu repository,
+    add as `development dependencies` on your `package.json` the following:
+
+      <code>
+
+         "devDependencies": {
+            "@polypoly-eu/podjs": "file:../../platform/podjs",
+         },
+
+      </code>
+
+2.  We use the `rollup` module bundler to build our features all together via the script
+
+    `$ rollup -c rollup.config.json`
+
+    which runs `rollup` and takes the `./src` directory as input and
+    puts the result in a `dist` folder. Those configurations need to be defined
+    in the `rollup.config.js` file.
+
+    We basically rely on our in-house `rollup-plugin-copy-watch` plugin
+    on top of the `Rollup` one which serves to copy files and folders with glob support
+    and also offers an additional watch to other sources than just Rollup's bundle content.
+    This is also needed to be installed as a dev dependency:
+
+    `$ npm i --save-dev ../../dev-utils/rollup-plugin-copy-watch`
+
+    So, we end up with a `package.json` file like this:
+
+         "devDependencies": {
+            "@polypoly-eu/rollup-plugin-copy-watch": "file:../../dev-utils/rollup-plugin-copy-watch",
+         },
+
+    So the feature uses the `@polypoly-eu/rollup-plugin-copy-watch"` plugin to watch the copied files.
+
+3.  Next step, you need to define your `rollup.config.js` at the same level as your `package.json`.
+
+    There you need to have the `copy` (of our `"@polypoly-eu/rollup-plugin-copy-watch"`) under the `plugins` section with the `targets`
```

Comment:
This is not actually *needed* per se. It's just a convenience. We use it because it's got the copy and the watch thing (I know, single responsibility principle) but people might want to use the superclass that just copies.